.github/workflows/keploy-ingest-llm.yml

```
@@ -0,0 +1,15 @@
+# keploy-ingest-llm.yml
+version: api/v1beta1
```

actionlint (1.7.7)

unexpected key "version" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)